doc/source/whatsnew/v1.0.0.rst

```diff
@@ -171,6 +171,7 @@ Plotting
 - Bug in :meth:`DataFrame.plot` producing incorrect legend markers when plotting multiple series on the same axis (:issue:`18222`)
 - Bug in :meth:`DataFrame.plot` when ``kind='box'`` and data contains datetime or timedelta data. These types are now automatically dropped (:issue:`22799`)
 - Bug in :meth:`DataFrame.plot.line` and :meth:`DataFrame.plot.area` produce wrong xlim in x-axis (:issue:`27686`, :issue:`25160`, :issue:`24784`)
+- Bug in :core.py:`BarPlot._decorate_ticks` xticks argument has no effect in bar plot (:issue:`14119`)
```

Comment:
We shouldn't refer to private methods in the release notes, which are for the user. Perhaps something like

```suggestion
- Bug in the ``xticks`` argument being ignored for :meth:`DataFrame.plot.bar` (:issue:`14119`)
```